src/app/app-routing.module.ts

```diff
@@ -2,20 +2,56 @@ import { NgModule } from "@angular/core";
 import { RouterModule } from "@angular/router";
 import { Route } from "@angular/router";
 import { AdminHomeComponent } from "./features/admin-home/components/admin-home.component";
+import { AdminSystemInformationComponent } from "./features/admin-system-information/components/admin-system-information.component";
```

Comment:
Remove the component imports as they are not used

Reply:
resolved